src/platform/silabs/rs911x/rsi_ble_config.h

```diff
@@ -89,7 +89,7 @@
 #define RSI_BLE_CUSTOM_SERVICE_UUID (0xFFF6)
 #define RSI_BLE_CUSTOM_LEVEL_UUID (0x1FF1)
 
-#ifdef RSI_M4_INTERFACE
+#if (defined(RSI_M4_INTERFACE) || defined(SLI_SI91X_MCU_INTERFACE))

```

Comment:
#else or #endif needs to be update here as well.

Reply:
updated comments in else and endif statements